Source/WebKit/WebProcess/WebPage/WebPage.messages.in

```diff
@@ -375,6 +375,11 @@ GenerateSyntheticEditingCommand(enum:uint8_t WebKit::SyntheticEditingCommandType
     DidConcludeDrop()
 #endif
 
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+    GetTextIndicatorForID(WTF::UUID uuid) -> (std::optional<WebCore::TextIndicatorData> textIndicator)
+    UpdateTextStyleVisibilityForID(WTF::UUID uuid, bool visible) -> ()
+# endif
```

Comment:
There's an extra space after the `#` here, which is causing the EWS builds to fail (since it can't find definitions for the subsequent IPC messages).